app/src/main/res/values/strings.xml

```diff
@@ -593,7 +593,7 @@
         search for similar issues before opening a new one. Any duplicated ticket will be labeled as duplicated and closed immediately.</p>
 
         <h5>• How can I get a PROMO CODE?</h5>
-        <p>If you are a student, you\'ll have to prove that to me. Proof can be sent via Email as listed below:</p>
+        <p>If you are a student, you\'ll have to prove it to me. Proof could be be sent via Email as listed below:</p>
```

Comment:
`Proofs`.

Reply:
Nope. We are using it as a general term.

Reply:
`A passport` is a `proof`. `A passport with a student card` are `proofs`.

Reply:
A passport is one type of proof. 
Google `proofs definition`

I don't even think `proofs` is a word...

Reply:
`Proofs` is a plural `proof`.

Reply:
Please link me to where that word exists, Proof is a noun.

http://www.dictionary.com/browse/proofs

Reply:
@dedepete ?